yolapy/configuration.py

```diff
@@ -0,0 +1,30 @@
+"""Configuration.
+
+Yolapy.configuration provides a key-value store used by the Yola client.
+
+Data is stored here in the module, benefits include:
+
+ * Configuration is decoupled from application logic.
+
+ * When instantiating multiple service models, each contains its own client.
+   This module allows for configuration to happen once then consumed multiple
+   times by each client.
+
+See README for example of use.
+
+"""
+config = {}
+_default = object()
+
+
+def configure(**kwargs):
+    """Save all keyword arguments as (key=value) configuration."""
+    config.update(kwargs)
+
+
+def get_config(key, default=_default):
```

Comment:
There's two different contexts for the concept of a default here which makes this a bit confusing for me. What about renaming `_default` to `_missing`?